File: bench/primes100k.c
#include <stdio.h>

int main() {
    int ret = 0;
    for (int at = 2; at < 100000; at++) {
        int is_prime = 1;
        for (int test = 2; test < at; test++) {
            if (at % test < 1) {
                is_prime = 0;
            }
        }
        if (is_prime) {
            ret = ret + 1;
        }
    }
    printf("%i\n", ret);
}

File: bench/primes100k.lua
local ret = 0
for at=2, 100000 do
    local is_prime = true
    for test=2, at-1 do
        if at % test < 1 then
            is_prime = false
        end
    end
    if is_prime then
        ret = ret + 1
    end
end
print(ret)